Phaser/phaser-workshop.adoc

```diff
@@ -65,31 +65,35 @@ function spelOpbouwen() {
 
     // Onze speler heeft grond nodig om op te lopen. Dat doen we hier
     grond = this.physics.add.staticGroup();
-    grond.create(400, 568, 'platform').setScale(2).refreshBody();
+    // De grond is 2x zo groot als een normaal platform
+    grond.create(400, 568, 'platform').setScale(2).refreshBody(); 
+
+    // We maken nog een aantal losse platforms
     grond.create(600, 400, 'platform');
     grond.create(50, 250, 'platform');
     grond.create(750, 220, 'platform');
 }
 ----
 
 Nu heb je de platforms gemaakt, controleer weer of het gelukt is.
+Als je het leuk vindt, kun je kijken wat er gebeurt als je de getallen bij de platforms aan past. Ook kun je nog een extra platform toevoegen als je wilt.
 
 
 == 3. Voeg het poppetje toe
 
-Voeg het volgende blok code to aan de 'spelOpbouwen' functie. Let op er staat al code in deze functie, voeg onderstaande toe aan wat er al staat. Dus direct onder grond.create(750, 220, 'platform');
+Voeg het volgende blok code to aan de 'spelOpbouwen' functie. Let op er staat al code in deze functie, voeg onderstaande toe aan wat er al staat. Dus direct onder `grond.create(750, 220, 'platform');`
 
 [source,js]
 ----
 // En nu voegen we de speler toe.
 speler = this.physics.add.sprite(100, 450, 'speler');
-// De speler mag uit het spel kunnen lopen!
+// De speler mag niet uit het spel kunnen lopen!
```

Comment:
Dit commentaar slaat verwarrend genoeg niet op regel 91 (die de stuiterbaarheid van de speler bepaalt), maar op regel 92.  Beter is denk ik end-line commentaren te gebruiken die aangeven wat elke regel van de volgende drie doet.